pkg/strategies/import.go

```diff
@@ -1,3 +1,6 @@
+// flag: export
```

Comment:
```suggestion
// flag: import
```

Reply:
Thanks for the suggestion. As i am still completely new to go, I'll have to catch up to pkg docs. 
This just came from using your package liking it, but taking some time to figure out which strategy I actually needed.